assets/translations/en.json

```
@@ -46,12 +48,15 @@
     "fees": "Fees",
     "recipientAddress": "Recipient address",
     "transactionHash": "Transaction Hash",
+    "transactionSuccessful": "Transaction Successful",
+    "transactionFailed": "Transaction Failed",
     "hash": "Hash",
     "fullHash": "Full Hash",
     "coinAddress": "Enter {} address",
     "youSend": "You are sending",
     "invalidAddress": "Invalid {} address",
     "customFeeCoin": "Custom fee [{}]",
+    "customNetworkFee": "Custom network fee",
```

Biome (1.9.4)

[error] The key customNetworkFee was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)